### PR TITLE
Change docker hub repository and update README

### PR DESCRIPTION
**Pull request by adityapb**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,23 +1,87 @@
...
+   .. table::
+
+    +----------------+-------------------------------+
+    |``-h, --help``  |Show help message              |
+    +----------------+-------------------------------+
+    |``--nodes``     |Number of nodes                |
+    +----------------+-------------------------------+
+    |``--ppn``       |Processors per node            |
+    +----------------+-------------------------------+
+    |``--walltime``  |Wall time                      |
```
Comment (katyhuff) [inline README.rst]: Let's make sure that this is a clear definition of the variable. "Wall time" is jargon. How about "max time your job can run, hours:minutes:seconds"

File: README.rst [hunk 2]
```diff
@@ -1,23 +1,87 @@
...
+    +----------------+-------------------------------+
+    |``-o``          |Output type ('sqlite' or 'h5') |
+    +----------------+-------------------------------+
+    |``--in-dir``    |Inputs directory               |
+    +----------------+-------------------------------+
+    |``--out-dir``   |Outputs directory              |
+    +----------------+-------------------------------+
+    |``--log-dir``   |Logs directory                 |
+    +----------------+-------------------------------+
+    |``--spec-file`` |Path of specification file     |
```
Comment (katyhuff) [inline README.rst]: Define the specification file.

File: README.rst [hunk 3]
```diff
@@ -1,23 +1,87 @@
...
+
+4. Submit the job.
 
    .. code-block:: bash
 
         $ qsub pbs_script.pbs
 
+Updating the docker image
+=========================
+
```
Comment (katyhuff) [inline README.rst:64]: Add some explanatory text here. Answer the question "when we might need to update the docker image in the future?"

File: README.rst [hunk 4]
```diff
@@ -1,23 +1,87 @@
 Running Cyclus on Blue Waters
 =============================
```
Comment (katyhuff) [inline README.rst:3]: Please add some explanatory text here. Specifically, set up an example use case. "Imagine you have a cyclus simulation that you would like to run many times with varied parameters or agents. You can use this workflow to generate an input file for each desired simulation and to run those input files simultaneously on the Blue Waters supercomputer."

File: README.rst [hunk 5]
```diff
@@ -1,23 +1,87 @@
...
+
+   Successfully built ``<IMAGE_ID>``
+
+2. Login to docker hub from command line,
+
+   .. code-block:: bash
+
+        $ sudo docker login --username=<USERNAME> --email=<EMAIL>
+
+3. Tag the image using the following command,
```
Comment (katyhuff) [inline README.rst]: Let's say, "Tag, or label, this version of the docker image using the following command:"

File: README.rst [hunk 6]
```diff
@@ -1,23 +1,87 @@
...
+
+   .. code-block:: bash
+
+        $ sudo docker tag <IMAGE_ID> arfc/cyclus_blue_waters:latest
```
Comment (katyhuff) [inline README.rst]: Tell the user how to decide what to put in the <IMAGE_ID> place. Explain what is a good example IMAGE_ID .

**Review by katyhuff — CHANGES_REQUESTED** [review]
This looks great! I left a few minor suggestions!

File: README.rst (hunk 5 above)
Comment (adityapb) [reply README.rst]: Now that we have the automated build set up, this won't be necessary

**Comment by adityapb** [comment]
I've made the changes @katyhuff

**Review by katyhuff — APPROVED** [review]
excellent.

**Pull request merged**